pages/homePage/homePage.css

```diff
@@ -0,0 +1,204 @@
+@import "/styles.css";
+
+.page-wrap {
+
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content'
+    ;
+}
+
+/* Header */
+.page-header {
+    grid-area: p_header;
+}
+
+.page-sidebar {
+    display: none;
+}
+
+.page-footer {
+    display: none;
+}
+
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    /* align-items: center; */
+    /* justify-content: center; */
```

Comment:
remove the commented code if it's not for future use

Reply:
+1